src/Microsoft.PowerShell.SDK/project.json

```diff
@@ -14,91 +14,91 @@
     "frameworks": {
         "net451": {},
         "netstandard1.6": {
-            "imports": ["dnxcore50", "portable-net45+win8"],
+            "imports": ["netcoreapp1.1", "portable-net45+win8"],
```

Comment:
Thanks for the quick response. Please add new commits to address comments, it's easier to review the changes this way 😄 I will squash the commits when merging the PR.